packages/react/src/lib/accessibility/Behaviors/Menu/menuItemBehavior.ts

```diff
@@ -43,12 +43,10 @@ const menuItemBehavior: Accessibility<MenuItemBehaviorProps> = props => ({
   },
 
   keyActions: {
-    root: {
+    wrapper: {
```

Comment:
we have one corner case where wrapper is removed by setting { wrapper: null } in the menuitem shorthand. I think enter/space would not work in that case. why does click on root close the menu but enter on root does not?

Reply:
> why does click on root close the menu but enter on root does not?

When Enter is pressed, `handleClick` is called on `MenuItem`. If root is used, the `this.props` in `handleClick` is not `MenuItemProps`, but instead something that looks like the root props. As a result of this, when `performClick` is called (from `handleClick`), the following branch of code is not executed (`this.props` does not contains `menu`):
```
    const { active, menu } = this.props

    if (menu) {
         ...
    }
    ...
```

Is it a correct behavior, that `this.props` is not set to `MenuItemProps` when we are using `root`?

Reply:
@lucivpav 
> Is it a correct behavior, that this.props is not set to MenuItemProps when we are using root and Enter?

I think it's not correct behavior. I would expect same menu item props to be passed

> but instead something that looks like the root props

What do you mean by that? What is the difference between those props objects?

Reply:
As discussed with Sofiya, if we are using `root`, the event does not propagate to other `root` elements, because they are siblings. Lets just use `wrapper` instead of `root`.

Reply:
as I mentioned - we have a case where wrapper is not used, this PR would break that case. Let's discuss in person tomorrow

Reply:
I have updated the PR. Note that if we are not using wrapper, then a menu cannot contain a submenu.